modules/core/shared/src/test/scala/freestyle/free/free.scala

```diff
@@ -203,12 +214,39 @@ class freeTests extends WordSpec with Matchers {
         def a: FS[Int]
         def b: FS.Par[Int] = (a, a).mapN(_+_)
       }
-      object Y extends X.Handler[Id] {
+      implicit object Y extends X.Handler[Id] {
         def a: Int = 5
       }
-      //X[X.Op].b.interpret[Id] shouldEqual 10
+      X.to[X.Op].b.interpret[Id] shouldEqual 10
     }
 
+    "using the Monad instance to combine operations into a FS.Seq" in {
+      import cats.syntax.monad._
```

Comment:
If it compiles like this, I don't think we need this.
For `flatMap` the import would be `import cats.syntax.flatMap._`.

It might now that it is `Free` which has a `flatMap` itself.